core/src/main/java/io/seata/core/store/db/LogStoreDataBaseDAO.java

```diff
@@ -67,6 +76,8 @@
 
     private String dbType;
 
+    private int transactionNameColumnSize;
```

Comment:
if some exception occur，it would be have a default value.

Reply:
fix done.